### PR TITLE
Added log to the service classes.

### PR DESCRIPTION
**Pull request by daniel-natali**
Added log information to the service classes

Ticket: jira...
Jenkins job: url..

### CODE REVIEW

File: src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java [hunk 1]
```diff
@@ -19,10 +19,14 @@ public ComponentHelper getHelper() {
   }
 
   public String getComponentDetails(String component) {
+	
+	  System.out.println("Entering component details method");
+	  System.out.println("Trying to find" + component + " details.");
```
Comment (danielnatali) [inline src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java]: Please use lombok instead. Avoid using system out.

File: src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java [hunk 2]
```diff
@@ -19,10 +19,14 @@ public ComponentHelper getHelper() {
...
+	  
     return helper.findComponentDetails(component);
   }
 
-  public void saveComponent(Component component) {
+  public void saveComponent(Component component) {	  
     System.out.println("Saving component");
```
Comment (danielnatali) [inline src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java]: Please use lombok for logging.

File: src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java [hunk 3]
```diff
@@ -13,10 +13,14 @@
   private StatusRepository statusRepository;
   
   public Iterable<Status> getStatusForAllComponents() {
+	  System.out.println("Getting status for all components..");
```
Comment (danielnatali) [inline src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java]: replace system out with lombok in all class.

File: src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java [hunk 4]
```diff
@@ -13,10 +13,14 @@
...
    return statusRepository.findAll();
  }
   
   public Status getStatusByComponent(String component) {
+	  
+	  System.out.println("Find status for" + component);
+	  
     return statusRepository.findByComponent(component);
   }
 }
```
Comment (danielnatali) [inline src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java]: Reformat class.

**Review by danielnatali — CHANGES_REQUESTED** [review]
Please review the comments.

File: src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java (hunk 1 above)
Comment (daniel-natali) [reply src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java]: Done!

File: src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java (hunk 2 above)
Comment (daniel-natali) [reply src/main/java/com/aurea/antipattern/sample/spring/services/ComponentService.java]: Done!

File: src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java (hunk 3 above)
Comment (daniel-natali) [reply src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java]: Done!

File: src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java (hunk 4 above)
Comment (daniel-natali) [reply src/main/java/com/aurea/antipattern/sample/spring/services/StatusService.java]: Done! spaces were removed

**Comment by daniel-natali** [comment]
Please @danielnatali review PR again.

**Review by danielnatali — APPROVED** [review]
Approved! Thanks!

**Pull request merged**